challenges/prototypes.js

```diff
@@ -6,28 +6,41 @@
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
 
+function CuboidMaker (length, width, height) {
+  this.length = length;
+  this.width = width;
+  this.height = height;
+}
```

Comment:
I really like this approach on your prototype. I guess I'm a bit biased, but alternatively, you could try and use one parameter instead and tag it to each one. Good work though.